UPGRADE-4.4.md

```diff
@@ -155,6 +160,16 @@ HttpKernel
    current directory or with a glob pattern. The fallback directories have never been advocated
    so you likely do not use those in any app based on the SF Standard or Flex edition.
  * Getting the container from a non-booted kernel is deprecated
+ * Marked the `AjaxDataCollector`, `ConfigDataCollector`, `EventDataCollector`,
+   `ExceptionDataCollector`, `LoggerDataCollector`, `MemoryDataCollector`,
+   `RequestDataCollector` and `TimeDataCollector` classes as `@final`.
+ * Marked the `RouterDataCollector::collect()` method as `@final`.
+ * Marked the `Profiler` class as `@final`.
+ * Deprecated the `ExceptionListener::logException()` method, use `logThrowable()` instead.
+ * The `DataCollectorInterface::collect()` method third parameter signature will
+   be `\Throwable $exception = null` instead of `\Exception $exception = null` in Symfony 5.0.
+ * The `HttpKernelInterface::handle()` method should throw a `\Throwable` instance instead of
```

Comment:
The description is very irritating. It CAN throw a Throwable. But as I explained in the other comment, it can throw a Throwable already now. The only difference is that Throwable didn't trigger the following Kernel events. I'm not sure if we should just catch throwable in 4.4 already and why it might be a bc break?
And even if we only do it in 5.0, I don't see what we should document here as people don't need to change anything.